packages/simple-hub/.vscode/launch.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Consider adding some of this to the monorepo root `launch.json`?

Reply:
https://github.com/statechannels/monorepo/issues/1083